arm-sql/compositeSql.json

```diff
@@ -4,7 +4,8 @@
     "description": "Composite Swagger for Azure Sql Database Client"
   },
   "documents": [
-    "./2014-04-01/swagger/sql.core.json",
     "./2014-04-01/swagger/firewallRules.json",
+    "./2014-04-01/swagger/replicationLinks.json",
+    "./2014-04-01/swagger/sql.core.json",
```

Comment:
Guess codeflow is back to not working.  #Resolved

---
In reply to: [85474480](https://github.com/Azure/azure-rest-api-specs/pull/643#discussion_r85474480) [](ancestors = 85474480)